src/umode.rs

```diff
@@ -413,9 +427,10 @@ impl UmodeTask {
                 .share_with_umode(data)
                 .map_err(Error::HypMap)?;
         }
-        let ret = task.run();
+        let ret = task.run(exec_ctx.attestation);
         // Errors encountered while executing the operation are unrecoverable: reset U-mode.
         if ret.is_err() {
+            println!("Umode error: {:?}", ret);
```

Comment:
leftover debug print?

Reply:
This prints the U-mode error (which is relatively rich in information) to help debugging in case of U-mode crash.